### PR TITLE
Document CLI vSphere Permission Configuration Invocation

### PR DESCRIPTION
**Pull request by jonathanmeier5**
*Issue #, if available:*

*Description of changes:*
Documents how to invoke programmatic commands for configuring permissions.

*Testing (if applicable):*
Manually configured users using both methods

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by abhinavmpandey08 — APPROVED** [review]
/approve

File: docs/content/en/docs/reference/vsphere/vsphere-preparation.md [hunk 1]
```diff
@@ -29,9 +25,102 @@ Follow these steps to create the user's vSphere folder:
...
+      - !!str "/MyDatacenter/vm/Templates/MyTemplates"
+```
+
+Set the admin credentials as environment variables:
+```bash
+export EKSA_VSPHERE_USERNAME=<ADMIN_VSPHERE_USERNAME>
+export EKSA_VSPHERE_PASSWORD=<ADMIN_VSPHERE_PASSWORD>
+```
+
+If the user does not already exists, you can create the user and all the specified group and role objects by runing:
```
Comment (chrisnegus) [inline docs/content/en/docs/reference/vsphere/vsphere-preparation.md:69]: Change "exists" to "exist.  Change "runing" to "running".

**Review by chrisnegus — COMMENTED** [review]
A couple of little typos. Otherwise looks good.

**Pull request merged**